jar-plugin/build.gradle

```diff
@@ -17,7 +17,14 @@ generateTestConfig {
 
 dependencies {
     compile project(':jruby-gradle-plugin')
-    compile 'com.github.jengelman.gradle.plugins:shadow:[1.2.2,2.0)'  // NEED TO FIX THIS OPEN END
+    /*
+     * NOTE: version 5.0.0 of the shadow plugin supports only Gradle 5.x and later
+     */
+    compile 'com.github.jengelman.gradle.plugins:shadow:4.0.4'
```

Comment:
If shadow a requirement for JRubyJar or an optional feature? If the latter, then I suggest using `compileOnly`, rather than `compile`.

Reply:
It is definitely a requirement